src/vm/arm/stubs.cpp

```diff
@@ -3767,8 +3772,23 @@ PCODE DynamicHelpers::CreateDictionaryLookupHelper(LoaderAllocator * pAllocator,
 {
     STANDARD_VM_CONTRACT;
```

Comment:
It would be nicer if we have an assertion check `pLookup->indirections == CORINFO_USEHELPER`, since optimization cases are not yet implemented.

Reply:
@wateret Thanks for your suggestion. I will make it!